src/main/java/net/sf/jabref/cli/SystemOutputPrinter.java

```diff
@@ -0,0 +1,21 @@
+package net.sf.jabref.cli;
+
+import net.sf.jabref.logic.importer.OutputPrinter;
+
+public class SystemOutputPrinter implements OutputPrinter {
+
+    @Override
+    public void setStatus(String s) {
+        System.out.println(s);
+    }
+
+    @Override
+    public void showMessage(Object message, String title, int msgType) {
```

Comment:
It can be safely replaced with String overall. While the showMessageDialog in JOption accepts an object as message param, the implementations in JabRef are always called with Strings. I did a quick check.